google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -185,22 +192,92 @@ public static TransportChannelProvider defaultTransportChannelProvider() {
         .setJwtEnabledScopes(JWT_ENABLED_SCOPES);
   }
 
-  /** Returns the object with the settings used for calls to ReadRows. */
+  /**
+   * Returns the object with the settings used for calls to ReadRows.
+   *
+   * <p>This is idempotent and streaming operation.
+   *
+   * <p>The idle timeout is configured via {@code ServerStreamingCallSettings#getIdleTimeout}. This
+   * timer will terminate any stream that has not has seen any demand or activity in the configured
+   * interval. To turn off idle checks, set the idleTimeout to {@link Duration#ZERO}.
+   *
+   * <p>Retries will be attempted if RPC failed with {@link Code#DEADLINE_EXCEEDED} and {@link
+   * Code#UNAVAILABLE} failure code.
+   *
+   * <p>RetryDelay starts with 100ms or 0.1 seconds, which multiplies with 1.3 times on every
+   * reattempt till a max retry timeout value of 60 seconds.
```

Comment:
Please drop `0.1 seconds`. Also, this isnt a timeout. Maybe reword to:

The RetryDelay between failed attempts starts with 100ms and exponentially increases by a factor of 1.3 until 60 seconds.

Reply:
Thanks for the detailed suggestion, I have updated these settings with the same.